package.json

```diff
@@ -67,7 +68,10 @@
     "typescript": "~3.9.7",
     "react-select": "^4.1.0",
     "react-dropzone": "^11.3.1",
-    "react-scroll": "^1.8.1"
+    "react-scroll": "^1.8.1",
+    "react-virtualized": "^9.22.3",
+    "react-select-async-paginate": "^0.5.3",
+    "react-highlight-words": "^0.17.0"
```

Comment:
Why are these libraries added to the `devDepencencies` and not the `depencencies` section? All libraries that are part of the frontend / webpack bundle must be available as `dependencies`. Otherwise these libraries are missing when the ordino package is used as dependency in an other repository.